sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -5186,100 +5189,6 @@ SUBROUTINE CLDRAD
 
       ENDIF         ! END OF LAEROPT IF-BLOCK
 
-!! Multiply by 1.E-6 to revert these fields back
-      IF (IGET(659)>0) THEN
```

Comment:
@zhanglikate With removal of the block of  output ID 659, do you think variable duem can be removed from this routine, and relevant routines ALLOCATE_ALL.f,  DEALLOCATE.f, INITPOST_GFS_NEMS_MPIIO.f, MPI_FIRST.f, VRBLS3D_mod.f, and parm/post_avblflds.xml?

Reply:
@zhanglikate Got it. Thanks!